src/guide/components/props.md

```diff
@@ -550,11 +550,11 @@ export default {
 
 </div>
 
-to validate that the value of the `author` prop was created with `new Person`.
+`author` propining qiymati `new Person` bilan yaratilganligini tasdiqlash uchun.
 
 ## Boolean Casting
 
-Props with `Boolean` type has special casting rules to mimic the behavior of native boolean attributes. Given a `<MyComponent>` with the following declaration:
+`Boolean` turidagi proplarda mahalliy boolean atributlarning xatti-harakatlariga taqlid qilish uchun maxsus kasting qoidalari mavjud. Quyidagi deklaratsiyaga ega `<MyComponent>` berilgan:
```

Comment:
Shu tarjimani to'g'risi tushunmadim, gruppadigilar bilan maslahatlashib ko'rish kerakmikan?

Reply:
maqul

Reply:
> Shu tarjimani to'g'risi tushunmadim, gruppadigilar bilan maslahatlashib ko'rish kerakmikan?

Bu nima boldi tarjima natijasi ?